src/ws-notification/ws-notification.js

```diff
@@ -94,7 +94,7 @@ export class WSNotification extends Component {
    * @returns {void}
    */
   animateIn(notification, index) {
-    const list = this.refs.list;
```

Comment:
I had to remove it to work with preact because the notification component used string refs which caused exceptions.